docs/api-guide/serializers.md

```diff
@@ -255,7 +255,8 @@ When passing data to a serializer instance, the unmodified data will be made ava
 
 ## Partial updates
 
-By default, serializers must be passed values for all required fields or they will raise validation errors. You can use the `partial` argument in order to allow partial updates.
+By default, serializers must be passed values for all required fields or they will raise validation errors. You can use the `partial` argument in order to allow partial updates. Note that `STRICT_PARTIAL_UPDATE` must be set to `True` to enable field specific updates using `.save(update_fields=[...])` for `ModelSerializer`
+with `partial=True`. `PARTIAL_UPDATE_EXTRA_FIELDS` lets you specify the fields you will like to update everytime like `mod_date` or `modified_at`.
```

Comment:
Does it make sense to have this "extra" as a global settings? I mean: different models have different fields and it is not guaranteed that _every_ model in a project shares a subset of field names.

IMHO it would be much better to have a serializer specific "Meta option".

Reply:
I agree. That would be a better option actually

Reply:
I will look into that and make some changes

Reply:
@sevdog I have updated the PR. Kindly take a look and let me know what you think. thanks